.eslintrc.yml

```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  no-console: off
```

Comment:
**Suggestion:** Try moving this to the `security-center/snippets` directory. That might resolve the ESlint issues.

Reply:
Ohhh, good catch. I noticed this and wondered if it should be migrated with everything else; but then it slipped my mind when this error came up. Thanks for pointing that out for me!